assets/questions/findChar.js

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for finding the
+ * First Non-Repeating Character in a String.
+ */
+
+globalData.questions['findChar'] = { // eslint-disable-line dot-notation
```

Comment:
This might need to be findFirstNonRepeatingCharacter -- findChar is a bit vague for an ID.